static/generate_i18n_resources_list.py

```diff
@@ -40,14 +41,21 @@ def get_translation_info(filepath):
             continue
         print("Processing", i18n_file.name)
         if i18n_file.name != "test.json":
-            lang_code, lang_name = get_translation_info(i18n_file)
+            translation_info = get_translation_info(i18n_file)
+            lang_name = translation_info[1]
             if lang_name:
-                language_list.append((lang_code, lang_name))
+                language_list.append(translation_info)
             else:
                 print(f"Warning: missing 'name' in {i18n_file.name}")
         f.write(str(i18n_file.relative_to(script_path.parent)) + '\n')
 
-language_list = [{name: code} for code, name in sorted(language_list)]
+def make_language_entry(lang_info_tuple):
```

Comment:
```suggestion
def make_language_entry(iso_code, self_name, translation_count):
```

and expand it when calling

```py
language_list = [make_language_entry(*lang_info) for lang_info in sorted(language_list)]
```